config/vufind/GeniePlus.ini

```diff
@@ -0,0 +1,77 @@
+[API]
+base_url = "https://your-api-base-url"
```

Comment:
Some comments for these would be nice, unless they're totally obvious for any GeniePlus user.

Reply:
Comments expanded.